### PR TITLE
[FrameworkBundle] Remove duplicated code in RouterDebugCommand

### PR DESCRIPTION
**Pull request by JhonnyL**
| Q | A |
| --- | --- |
| Branch? | 2.7 |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

**Comment by sstok** [comment]
👍 

Status: reviewed

**Comment by hhamon** [comment]
Good catch ;)

File: src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php [hunk 1]
```diff
@@ -83,9 +83,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $name = $input->getArgument('name');
         $helper = new DescriptorHelper();
+        $routes = $this->getContainer()->get('router')->getRouteCollection();
 
         if ($name) {
-            $route = $this->getContainer()->get('router')->getRouteCollection()->get($name);
+            $route = $routes->get($name);
             if (!$route) {
```
Comment (hhamon) [inline src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php]: you can even simplify to `if (!$route = $routes->get($name)) {`

**Comment by fabpot** [comment]
Thank you @JhonnyL.

**Pull request merged**